tests/unit/s2n_client_signature_algorithms_extension_test.c

```diff
@@ -81,6 +81,7 @@ int main(int argc, char **argv)
         EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_SERVER));
         POSIX_GUARD(s2n_connection_set_config(conn, config));
         conn->actual_protocol_version = S2N_TLS12;
+        conn->secure->cipher_suite = &s2n_ecdhe_rsa_with_aes_256_gcm_sha384;
 
```

Comment:
The "null" cipher suite "uses" RSA kex: https://github.com/aws/s2n-tls/blob/795f4dc65335b56dfb356d6eaac9429fbae5b75b/tls/s2n_cipher_suites.c#L210-L218 So I explicitly set a cipher suite.